enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -446,7 +447,13 @@ def signal_kernel(self, signum):
         else:
             raise RuntimeError("Cannot signal kernel. No kernel is running!")
 
-    def cleanup(self, connection_file=True):
+    def cleanup(self, restart=False):
+        """Clean up resources when the kernel is shut down"""
+        warnings.warn("Method cleanup(connection_file=True) is deprecated, use cleanup_resources(restart=False).",
+                      DeprecationWarning)
```

Comment:
Could you please update this to `FutureWarning` since deprecation warnings aren't produced when used from this location?

Reply:
Fixed